app/controllers/telegram/webhook_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Telegram::WebhookController < Telegram::Bot::UpdatesController
+  def message(message)
+    chat_id = message['chat']['id']
+    user = User.find_by(chat_id: chat_id)
+    unless user
+      user = User.new(chat_id: chat_id)
+      user.save!
+    end
+    user.respond_feedback(answer: message['text'])
```

Comment:
To be honest, I think the `respond_feedback` is a bit opaque (at least, I had to look it up to understand what exactly it does.) I’d prefer something like this:

```ruby
Issue.last.add_reply(user: user, reply: message['text']
```